mk/config.mk

```diff
@@ -548,6 +548,11 @@ endif
 # an exploration based on compatible drivers found. It is default disabled.
 CFG_DRIVERS_DT_RECURSIVE_PROBE ?= n
 
+# When enabled, CFG_SCMI_MSG_USE_CLK will embed SCMI clocks function using
+# the clock framework and define plat_scmi_clock* functions.
```

Comment:
s/plat_scmi_clock*/plat_scmi_clock_*()/

Suggestion:
```
# When enabled, CFG_SCMI_MSG_USE_CLK embeds SCMI clocks
# registering services for the platform SCMI server and implements
# the platform plat_scmi_clock_*() functions.
```

Last, I would move these around line 800 with other `CFG_SCMI_MSG_*` switches.

Reply:
Acked, I'll move it under `CFG_SCMI_MSG_DRIVERS`